core/src/epicli/data/common/defaults/configuration/feature-mapping.yml

```diff
@@ -9,39 +9,39 @@ specification:
     - name: kubernetes-node
       enabled: yes
     - name: elasticsearch
-      enabled: yes
+      enabled: no
```

Comment:
Shouldnt these be turned back on to yes?

Reply:
It should, will fix it